src/Resources/config/doctrine/CreditMemo.orm.xml

```diff
@@ -10,7 +10,7 @@
         <id name="id" column="id" type="string" />
 
         <field name="number" />
-        <field name="orderNumber" />
+        <field name="orderNumber" column="order_number" />
         <field name="total" type="integer" />
         <field name="units" type="json" />
         <field name="currencyCode" />
```

Comment:
To make it consistent, we can also change this field column to `currency_code` 🐎

Reply:
Yeah, I did that now :)